.github/workflows/test_openvino_basic.yml

```diff
@@ -0,0 +1,49 @@
+name: OpenVINO - Basic Test
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '41 1 * * *'  # run every day at 1:41
+  push:
+    paths:
+    - 'tests/openvino/test_modeling_basic.py'
+    - '.github/workflows/test_openvino_basic.yml'
+  pull_request:
+    paths:
+    - 'tests/openvino/test_modeling_basic.py'
+    - '.github/workflows/test_openvino_basic.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.10"]
+        transformers: ['transformers', 'git+https://github.com/huggingface/transformers.git']
+        optimum: ['optimum', 'git+https://github.com/huggingface/optimum.git']
+
+    runs-on: ubuntu-20.04
```

Comment:
Python 3.8 comes by default with ubuntu-20.04. I wonder why we are going to check `3.7` and `3.10`?

Reply:
I find it useful to have lower and upper bound tests. The main tests use 3.8 and 3.9, so having 3.7 and 3.10 here would be complimentary to that (we could probably also upgrade the main test to use Python 3.10, since that's the default in Ubuntu 22.04 and now supported by OpenVINO). It is unlikely that tests would fail in 3.8 but work fine in both 3.7 and 3.10. I can add 3.8 here, I didn't include it because it seemed a bit over-enthusiastic to test 3 python versions and I would like to test 3.7 too. One reason for the test to fail is that dependencies of dependencies suddenly drop support for lower Python versions in their new releases, and having the lowest supported version in the automatic tests alerts us to that.

Reply:
Ok, now it makes sense. Placing a comment about that could be helpful.

Reply:
Thanks, I added a comment.